cypress/integration/data-import/view-all-filter.spec.js

```diff
@@ -0,0 +1,104 @@
+import DataImportViewAllPage from '../../support/fragments/data_import/dataImportViewAllPage';
+import fileManager from '../../support/utils/fileManager';
+import DateTools from '../../support/utils/dateTools';
+import { Accordion } from '../../../interactors';
+import getRandomPostfix from '../../support/utils/stringTools';
+
+describe('ui-data-import: Filter the "View all" log screen', () => {
+  // Path to static file in fixtures
+  const pathToStaticFile = 'oneMarcBib.mrc';
+  // Create unique names for MARC files
+  const fileNameForFailedImport = `test${getRandomPostfix()}.mrc`;
+  const fileNameForSuccessfulImport = `test${getRandomPostfix()}.mrc`;
+
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+    cy.getToken(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+
+    // Create files dynamically with given name and content in fixtures
+    fileManager.createFile(`cypress/fixtures/${fileNameForFailedImport}`);
+    // read contents of static file in fixtures
+    cy.readFile(`cypress/fixtures/${pathToStaticFile}`).then(content => {
+    // and write its contents to the file which runs successfully and create it
+      fileManager.createFile(`cypress/fixtures/${fileNameForSuccessfulImport}`, content);
+    });
+
+    // Upload files
+    // runs with errors
+    cy.uploadFile(fileNameForFailedImport);
```

Comment:
there is double cy.visit(). Could you try to move it from cy.upload into the test?